ArchiSteamFarm/Bot.cs

```diff
@@ -635,6 +685,27 @@ internal sealed class Bot : IDisposable {
 			return await ArchiWebHandler.GetTradeHoldDurationForTrade(tradeID).ConfigureAwait(false);
 		}
 
+		internal async Task<(Dictionary<string, string> Used, Dictionary<string, string> Unused)> GetUsedAndUnusedKeys() {
+			Task<Dictionary<string, string>> task1 = GetKeysFromFile(KeysToRedeemUsedFilePath);
+			Task<Dictionary<string, string>> task2 = GetKeysFromFile(KeysToRedeemUnusedFilePath);
```

Comment:
Actually now I noticed that this won't be lazily evaluated so it won't work if you leave it like that. Use something like `tasks = new string[] { KeysToRedeemUsedFilePath, KeysToRedeemUnusedFilePath }.Select(file => GetKeysFromFile(file)`.

Reply:
revert my changes?